private-templates-service/adaptivecards-templating-service/src/tests/router.test.ts

```diff
@@ -8,36 +8,34 @@ import bodyParser from "body-parser";
 import { InMemoryDBProvider } from "../storageproviders/InMemoryDBProvider";
 import { ITemplate, ITemplateInstance } from "../models/models";
 
-export default async function getToken(): Promise<string> {
-    const request = require('request-promise');
-    const endpoint = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/oauth2/token";
-    const requestParams = {
-      grant_type: "client_credentials",
-      client_id: "4803f66a-136d-4155-a51e-6d98400d5506",
-      client_secret: "#{secrets.AZURE_ADAPTIVECMS_CLIENT_SECRET}#",
-      resource: "https://graph.windows.net"
-    };
-    return await request.post({ url: endpoint, form: requestParams })
-      // put in try catch
-      .then((err: any, response: any, body: any) => {
-        if (err) {
-          let parsedBody = JSON.parse(err);
-          return Promise.resolve(parsedBody.access_token);
-        }
-        else {
-          console.log("Body=" + body);
-          let parsedBody = JSON.parse(body);
-          if (parsedBody.error_description) {
-            console.log("Error=" + parsedBody.error_description);
-            return Promise.resolve("hi");
-          }
-          else {
-            console.log("Access Token=" + parsedBody.access_token);
-            return Promise.resolve(parsedBody.access_token);
-          }
-        }
-      });
-  }
+export default async function getToken(): Promise<string> {
+  const request = require("request-promise");
+  const endpoint = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/oauth2/token";
```

Comment:
From config?

Reply:
yup, a lil later gonna be from config :))